substrate/frame/identity/src/lib.rs

```diff
@@ -265,33 +297,47 @@ pub mod pallet {
 
 	/// A map of the accounts who are authorized to grant usernames.
 	#[pallet::storage]
-	pub type UsernameAuthorities<T: Config> =
-		StorageMap<_, Twox64Concat, T::AccountId, AuthorityPropertiesOf<T>, OptionQuery>;
+	pub type AuthorityOf<T: Config> =
```

Comment:
This implies for a given suffix, there can be only one authority? Wasn't clear from the docs.

Reply:
Yes, one authority per suffix. I will update the docs.